### ISSUE TITLE
[OJS] Restricted list with reviewers

### ISSUE THREAD

**Issue by GrazingScientist** [pkp/pkp-lib#5009]
In our institute came up the issue that sometime guest editors are invited. With the european data privacy policy, this raises the problem that a guest editor can see ALL reviewers and not only those that are relevant for the given issue.

Would it be possible for the core editor team to make a list of reviewers an editor can see?

Thanks a lot! 😄

**Comment by asmecher** [comment]
@GrazingScientist, are you asking about restricting the ability of guest editors to view reviewer information for submissions that the guest editor is explicitly assigned to?

**Comment by GrazingScientist** [comment]
Yes, so the guest editor is assigned to an issue. But the editing staff do not want them to see ALL possible reviewers but only a specific list (e.g. chosen by interest or by manually picking them in the system).
